sql/src/main/java/io/crate/exceptions/Exceptions.java

```diff
@@ -53,15 +54,19 @@
 
     private final static ESLogger LOGGER = Loggers.getLogger(Exceptions.class);
     private final static Predicate<Throwable> EXCEPTIONS_TO_UNWRAP = throwable ->
-        throwable instanceof RemoteTransportException ||
+        throwable instanceof TransportException ||
```

Comment:
Shouldn't we keep the RemoteTransportException in there as well? Or add ElasticsearchWrapperException